migrations/versions/74cdf9a8b9_.py

```diff
@@ -0,0 +1,268 @@
+    sa.PrimaryKeyConstraint('id'),
+    sa.UniqueConstraint('topic_number'),
+    sa.UniqueConstraint('url')
+    )
+    op.create_table('roles_users',
+    sa.Column('user_id', sa.Integer(), nullable=True),
+    sa.Column('role_id', sa.Integer(), nullable=True),
+    sa.ForeignKeyConstraint(['role_id'], ['roles.id'], ),
+    sa.ForeignKeyConstraint(['user_id'], ['users.id'], )
+    )
+    op.create_table('connections',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('user_id', sa.Integer(), nullable=False),
+    sa.Column('provider_id', sa.String(length=255), nullable=True),
+    sa.Column('provider_user_id', sa.String(length=255), nullable=True),
+    sa.Column('access_token', sa.String(length=255), nullable=True),
+    sa.Column('secret', sa.String(length=255), nullable=True),
+    sa.Column('display_name', sa.String(length=255), nullable=True),
+    sa.Column('full_name', sa.String(length=255), nullable=True),
+    sa.Column('profile_url', sa.String(length=512), nullable=True),
+    sa.Column('image_url', sa.String(length=512), nullable=True),
+    sa.Column('rank', sa.Integer(), nullable=True),
+    sa.ForeignKeyConstraint(['user_id'], ['users.id'], ),
+    sa.PrimaryKeyConstraint('id')
+    )
+    op.create_table('proposals',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('name', sa.String(length=255), nullable=True),
+    sa.Column('description', sa.Text(), nullable=True),
+    sa.Column('sbir_topic_reference', sa.String(length=255), nullable=True),
+    sa.Column('start_date', sa.DateTime(timezone=True), nullable=True),
+    sa.Column('end_date', sa.DateTime(timezone=True), nullable=True),
+    sa.Column('created_at', sa.DateTime(timezone=True), nullable=False),
+    sa.Column('updated_at', sa.DateTime(timezone=True), nullable=False),
+    sa.Column('organization_id', sa.Integer(), nullable=True),
+    sa.ForeignKeyConstraint(['organization_id'], ['organizations.id'], onupdate='CASCADE', ondelete='RESTRICT'),
+    sa.PrimaryKeyConstraint('id')
+    )
+    op.create_table('documents',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('name', sa.String(length=255), nullable=True),
+    sa.Column('description', sa.Text(), nullable=True),
+    sa.Column('filepath', sa.String(length=255), nullable=False),
+    sa.Column('created_at', sa.DateTime(timezone=True), nullable=False),
+    sa.Column('updated_at', sa.DateTime(timezone=True), nullable=False),
+    sa.Column('organization_id', sa.Integer(), nullable=True),
+    sa.ForeignKeyConstraint(['organization_id'], ['organizations.id'], onupdate='CASCADE', ondelete='RESTRICT'),
+    sa.PrimaryKeyConstraint('id')
+    )
+    op.create_table('phases',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('phase', sa.Text(), nullable=False),
+    sa.Column('topic_id', sa.Integer(), nullable=False),
+    sa.ForeignKeyConstraint(['topic_id'], ['topics.id'], ),
+    sa.PrimaryKeyConstraint('id'),
+    sa.UniqueConstraint('phase')
```

Comment:
I don't think we should require unique text for phases. Some of them (esp. Phase III) are closer to boilerplate language.